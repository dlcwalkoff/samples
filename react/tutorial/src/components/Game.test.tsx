import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React = require('react');
import { Game } from './Game';

describe('Game', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Game />
    );

    expect(shallowToJson(output)).toMatchSnapshot();
  });
});