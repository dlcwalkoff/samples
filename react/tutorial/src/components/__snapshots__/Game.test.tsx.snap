// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game should render correctly 1`] = `
<div
  className="game"
>
  <div
    className="game-board"
  >
    <Board />
  </div>
  <div
    className="game-info"
  >
    <div />
    <ol />
  </div>
</div>
`;
